### DISCUSSION TITLE
Unable to load project after updating to 2.0.14

### DISCUSSION THREAD

**Discussion by AshleyThorntonWWT** [opengisch/QField#2802] [Q&A]
Hi all!

We've been using Qfield for a project over the last few months and have recently run into an issue.

I think that we hadn't been regularly updating Qfield and we were instead looking for the survey season to finish before we did so, but we accidentally updated Qfield to its newest version last week.  (2.0.14)

Unfortunately, when we now try to open the project on Android we're given the message: "The following layers could not be loaded, please review those and reconfigure the QGIS project."
The files listed are the base_map.tif, and then all the proceeding shapefiles.shp for each week that we carry out and record the survey.

When we access the Qfield data, saved on the android, via the desktop's QGIS we get the error messages seen in the image attached.

Things were okay before the update, so we're unsure what happened.

Any help would be greatly appreciated!

Best wishes,
Ash

![image](https://user-images.githubusercontent.com/104214582/164743809-cf207486-7c23-40e5-96dd-2f155e02efe2.png)

**Comment by nirvn** [comment]
@AshleyThorntonWWT , QField 2.0 had to significantly change the way it accesses storage on Android, please read this documentation: https://docs.qfield.org/get-started/storage/

That should give you a good idea as to how to import your project currently stored on your device in a location I assume QField has no right to access anymore.

As for QGIS desktop error, try to copy your whole project folder off your device onto your desktop, make sure your datasets are present, and open it from there. Sometimes your device might connect via USB in a different mode than "raw" storage access which could cause trouble.

**Reply by AshleyThorntonWWT** [reply]
Hello!

Thank you so much for your help, Mathieu. Instantly it's fixed and we can carry on with what we were doing, we thought all was lost!

Thank you again, and have a great day. You're a lifesaver!
